### DISCUSSION TITLE
✅ 🚀 Feature Request: Show more than 1 taxonomy badge on Posts cards skin

### DISCUSSION THREAD

**Discussion by LiesBr** [elementor/elementor#22548] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

The current cards skin in the posts widget only shows 1 taxonomy term. Even though the post might have several

### Describe the solution you'd like

I would like the option to choose a number of terms to display. 

### Describe alternatives you've considered

I realise in the standard skin, there isnt much room for more, but I figure: IF you want more badges, you are naturally going to make room for them. I.e., by making the image larger, or by moving the badges, or adapting font-size.. etc.

**Comment by nicholaszein** [comment]
Hello there @LiesBr!

### ✅ This feature is already available with the new Loop Grid widget. 

You can build your Cards the way you want, and add multiple post terms using the Post Terms Dynamic Tag:
https://elementor.com/help/dynamic-tags-pro/#:~:text=Post%20Terms%20%E2%80%93%20The%20chosen%20taxonomy%20for%20the%20current%20post%20(category%2C%20tags%2C%20etc.)

Please, use the Loop Grid widget if you want more control over your Post designs.
https://elementor.com/help/loop-grid/

Kind regards
